src/sentry/static/sentry/app/views/settings/account/accountSecurity/index.jsx

```diff
@@ -11,6 +11,8 @@ import AsyncView from 'app/views/asyncView';
 import Button from 'app/components/button';
 import CircleIndicator from 'app/components/circleIndicator';
 import EmptyMessage from 'app/views/settings/components/emptyMessage';
+import FieldLabel from 'app/views/settings/components/forms/field/fieldLabel.jsx';
```

Comment:
we generally don't include file exts for `js/jsx` (we need to add a lint rule for this).